sdk/communication/Azure.Communication.CallingServer/api/Azure.Communication.CallingServer.netstandard2.0.cs

```diff
@@ -34,10 +34,10 @@ public partial class CallConnection
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Communication.CallingServer.CallConnectionProperties>> GetPropertiesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response Hangup(bool forEveryone, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> HangupAsync(bool forEveryone, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response PlayMedia(Azure.Communication.CallingServer.PlaySource playSource, System.Collections.Generic.IEnumerable<Azure.Communication.CommunicationIdentifier> playTo, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response> PlayMediaAsync(Azure.Communication.CallingServer.PlaySource playSource, System.Collections.Generic.IEnumerable<Azure.Communication.CommunicationIdentifier> playTo, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response PlayMediaToAll(Azure.Communication.CallingServer.PlaySource playSource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response> PlayMediaToAllAsync(Azure.Communication.CallingServer.PlaySource playSource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response Play(Azure.Communication.CallingServer.FileSource fileSource, System.Collections.Generic.IEnumerable<Azure.Communication.CommunicationIdentifier> playTo, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
For this section, I thought you wanted to accept PlaySource? for accepting different types of sources (FileSource, TTS-Source, etc)

Reply:
Hey @minwoolee-msft, happy to discuss this: We received the comment in the reviews that PlaySource naming is not great, and some others were confused with what we are actually playing.
I thought that having these methods help to the reading, in the semantics of "Play this FileSource", "Play this TextSource", etc. So we can cut on the naming problems. 
We are doing something similar on download with `downloadTo(String filePath...)` and `downloadTo(Stream stream, ...` and I think that it helps.

Please share your thoughts on this :)

Reply:
From API, I see both Playsource and filesource - the playsource has PlaySourceType (currenlty only File is there), and it has FileSource as one of its properties. So my assumption here was that PlaySource is the wrapper model (abstract) and the contoso would pass in FileSource, or other source to it. 
I am afraid if we do pass just the filesource here, what is the purpose of having playsource in the API? can we just drop playsource?

Reply:
Well, `PlaySource` will become useful when we add more sources to the mix. 
Maybe I can move it to be an internal object, so it works to refactor code in the future.